meson.build

```diff
@@ -371,74 +371,44 @@ if not (pytorch_support_is_available and caffe2_support_is_available)
   endif
 endif
 
-# Python
-have_python2 = false
+# Python3
 have_python3 = false
-
-pg_pkgconfig = find_program('pkg-config')
-
-# Check python 2.7
-python2_dep = dependency('python-2.7', required: get_option('python2-support'))
-if python2_dep.found()
-
-  python2_inc_args = []
-  python2_inc_args += run_command(pg_pkgconfig, ['python-2.7', '--cflags']).stdout().strip().split()
-  python2_inc_args += run_command('python2.7', ['-c', 'import site\nfor i in site.getsitepackages(): print("-I" + i + "/numpy/core/include")']).stdout().strip().split()
-  python2_inc_args += '-I' + run_command('python2.7', ['-m', 'site', '--user-site']).stdout().strip() + '/numpy/core/include'
-  python2_inc_valid_args = []
-
-  foreach python2_inc_arg : python2_inc_args
-    if cxx.has_argument(python2_inc_arg) and \
-        cxx.check_header('numpy/arrayobject.h', args: python2_inc_arg, dependencies : python2_dep)
-      python2_inc_valid_args += python2_inc_arg
-      have_python2 = true
-      break
-    endif
-  endforeach
-
-  if have_python2
-    python2_deps = declare_dependency(dependencies: python2_dep,
-        compile_args : python2_inc_valid_args)
-  else
-    warning('Found python2.7, but failed to find numpy.')
-    warning('Disable nnstreamer-python2.')
-  endif
-
-else
-  warning('Python2.7 either not found or disabled')
-endif
-
-# Check python 3.x
-python3_dep = dependency('python3', required: get_option('python3-support'))
-if python3_dep.found()
-  if python3_dep.version().version_compare('>= 3.8')
+if not get_option('python3-support').disabled()
```

Comment:
Note that if "python3-support" is enabled and python3 is not found --> BUILD ERROR!
if "python3-support" is auto and python3 is not found --> Just disable python3. (no error)

Your code appears to handle both equally.

Reply:
Applied, thanks!